code/modules/projectiles/projectile.dm

```diff
@@ -234,7 +234,7 @@
 	if(!check_faction(A))
 		return FALSE 	 //Nsv13 - faction checking for overmaps. We're gonna just cut off real early and save some math if the IFF doesn't check out.
 
-	var/datum/point/pcache = trajectory.copy_to()
+	var/datum/point/pcache = trajectory?.copy_to()
```

Comment:
Will this error later in the function when other things try to use pcache?

Reply:
Probably but I don't want to think too hard about it

Reply:
IT'LL STILL RUNTIME THOUGH, BAD
Either make a default trajectory or return early, I think

Reply:
no please I don't want to work with byond

Reply:
You can literally set it up with a semi colon or something to force a alternative instead I think.